app/pages/DateOfBirthPage.scala

```diff
@@ -32,7 +32,7 @@ case object DateOfBirthPage extends QuestionPage[LocalDate] {
   override protected def nextPageNormalMode(answers: UserAnswers): Call =
     answers.get(HasNinoPage) match {
       case Some(true)  => routes.IndexController.onPageLoad()
-      case Some(false) => routes.UkAddressController.onPageLoad(NormalMode)
+      case Some(false) => routes.AddressInUkController.onPageLoad(NormalMode)
```

Comment:
Please check because this does not make sense. If you have a NINO then you are in the UK so no need to ask and you can request the UK address.